documentation/misc/zookeeper-manual-scaling-methods.md

```diff
@@ -0,0 +1,149 @@
+# Zookeeper 3.5.x Manual Scaling Procedures
+
+Zookeeper 3.5.x uses a new configuration procedure compare to 3.4.x servers. This 
+[dynamic reconfiguration](https://zookeeper.apache.org/doc/r3.5.7/zookeeperReconfig.html) means that servers need to be
+added and removed via the Zookeeper command line client (or Admin API) in order to properly maintain a stable Zookeeper 
+cluster. 
+
+The procedures below outline the steps required to scale up (add) and scale down (remove) servers from an existing 
+Strimzi Kafka deployment.
+
+## Scale up procedure
+
+The cluster should be scaled **one server at a time**. 
+
+1) Create an initial Strimzi cluster with n=3 replicas (3 Kafka + 3 ZK servers).
+
+2) Set the replica count to n=4 in the zookeeper section of the Kafka CR.
+
+3) Allow the ZK server (zookeeper-3) to start up normally and establish a link to the existing quorum.
+
+This can be checked with the following command:
+
+```
+kubectl exec -n myproject -it my-cluster-zookeeper-3 -c zookeeper -- bash -c "echo 'srvr' | nc 127.0.0.1 21813 | grep 'Mode:'"
```

Comment:
This needs some more explaining about the port name (why `21813` and how does it change for other ports.)

Reply:
You should also be more generic about the namespace (myproject) and the cluster name (my-cluster) I think.